src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -231,7 +231,7 @@ sockinfo_tcp::sockinfo_tcp(int fd):
 	m_iomux_ready_fd_array = NULL;
 
 	/* SNDBUF accounting */
-	m_sndbuff_max = 0;
+	m_sndbuff_max = safe_mce_sys().sysctl_reader.get_tcp_wmem()->default_value;
```

Comment:
Do you see conflict in calculation of this value with https://github.com/Mellanox/libvma/blob/master/src/vma/sock/sockinfo_tcp.cpp#L3508